feat(command-dev): support secure https development (#2132)
netlify/cli: docs/netlify-dev.md

  jwtSecret = "secret" # The secret used to verify tokens for JWT based redirects
  jwtRolePath = "app_metadata.authorization.roles" # Object path we should look for role values for JWT based redirects
  autoLaunch = true # a Boolean value that determines if Netlify Dev launches the local server address in your browser
  # to start an https server instead of an http one, configure a certificate and key files
  [dev.https]

>     certFile = "cert.pem" # path to the certificate file

Author: This is for consistency.

See https://github.com/netlify/cli/issues/2127

For now, it is important to include instructions to create addons for each member of your team, as there is no way to specify addons inside of `netlify.toml`. We are working on this.

### Running an https server for local development

> By default, `netlify dev` starts an http server. If you require an https one, you can configure a certificate and key file to be used by `netlify dev`.

Reviewer: minor suggestions:

```suggestion
By default, `netlify dev` starts an HTTP server. If you require HTTPS, you can configure a certificate and key file for use by `netlify dev`.
```

Example `netlify.toml`:

```toml
[dev]
[dev.https]
certFile = "cert.pem"
keyFile = "key.pem"
```

> **Important -** if you're using a self signed certificate you might need to configure your browser to accept it when running on `localhost`.
> For Chrome you can do it by visiting the following URL in your browser `chrome://flags/#allow-insecure-localhost`.

Reviewer: Minor suggestions:

```suggestion
**Important -** If you're using a self-signed certificate, you might need to configure your browser to accept it when running on `localhost`.
To enable this setting for Chrome, visit `chrome://flags/#allow-insecure-localhost` in your browser.
```
